public-event-list.md

```diff
@@ -136,268 +138,566 @@ To get the custom included in the Public Event List payload, first create a Fiel
-          "name": "Falcon Theatre, Oct. 5, 2012",
-          "eventId": "a018000000N2wwSAAR",
-          "venueId": "a05800000070JMaAAM",
-          "soldOut": false,
-          "seatingType": "Pick Your Own Seats",
-          "saleStatus": "On Sale",
-          "purchaseUrl": "https://sillytickets.secure.force.com/ticket#sections_a068000000OjDHlAAN",
-          "noSaleMessage": null,
-          "formattedDates": {
-            "YYYYMMDD": "20121006",
-            "LONG_MONTH_DAY_YEAR": "October 6, 2012",
-            "ISO8601": "2012-10-06T00:00:00.000Z"
-          },
-          "detail": null,
-          "allocations": [
-            {
-              "id": "a078000000Gh8mCAAR",
-              "name": "Side",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 0,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3kAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8mCAAR",
-                  "sortOrder": null,
-                  "price": 39.5,
-                  "fee": 4
-                }
-              ]
-            },
-            {
-              "id": "a078000000Gh8mDAAR",
-              "name": "Center",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 1,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3lAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8mDAAR",
-                  "sortOrder": null,
-                  "price": 42,
-                  "fee": 4
-                }
-              ]
-            },
-            {
-              "id": "a078000000Gh8mFAAR",
-              "name": "Accessible Side",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 3,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3nAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8mFAAR",
-                  "sortOrder": null,
-                  "price": 39.5,
-                  "fee": 4
-                }
-              ]
-            },
-            {
-              "id": "a078000000Gh8oIAAR",
-              "name": "Accessible",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 4,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3oAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8oIAAR",
-                  "sortOrder": null,
-                  "price": 42,
-                  "fee": 4
-                }
-              ]
-            }
-          ]
-        }
-      ]
+  "events" : [ {
+    "type" : "Tickets",
+    "sortOrder" : 1,
+    "smallImagePath" : null,
```

Comment:
let's take out the `smallImagePath` and other fields that we're not documenting from the sample